docs/core/compatibility/toc.yml

```diff
@@ -8,18 +8,20 @@
     href: breaking-changes.md
   - name: By version
     items:
-    - name: .NET Framework to .NET Core
-      href: framework-core.md
-    - name: 2.2 to 3.0
-      href: 2.2-3.0.md
-    - name: 3.0 Preview 6 to 3.0 Preview 7
-      href: 3.0.6-3.0.7.md
-    - name: 3.0 Preview 7 to 3.0 Preview 8
-      href: 3.0.7-3.0.8.md
-    - name: 3.0 Preview 8 to 3.0 Preview 9
-      href: 3.0.8-3.0.9.md
+    - name: 3.0 to 3.1
+      href: 3.0-3.1.md
     - name: 3.0 Preview 9 to 3.0 RC1
```

Comment:
Are there no breaking changes from 3.0 RC1 to 3.0 GA? If not maybe we should make this entry preview 9 to 3.0 so it won't look like we've left a gap in the list.

Reply:
I changed it to just 3.0, but have further questions about versions that I sent in an email. Thanks for the review, I'm going to merge this now.